benchmarks/infra/stage-2/sample-tfvars/jetstream-sample.tfvars

```diff
@@ -0,0 +1,27 @@
+# can be obtained from stage-1 by running:
+# terraform output -json  | jq '."fleet_host".value'
+credentials_config = {
+  fleet_host = "https://connectgateway.googleapis.com/v1/projects/$PROJECT_NUMBER/locations/global/gkeMemberships/ai-tpu-benchmark"
+}
+
+# can be obtained from stage-1 by running:
+# terraform output -json  | jq '."project_id".value'
+project_id = "PROJECT_ID"
+
+bucket_name = "${PROJECT_ID}-tpu-model-repo-bucket-01"
+bucket_location = "US"
+
+output_bucket_name     = "${PROJECT_ID}-tpu-benchmark-output-bucket-01"
```

Comment:
Can we keep all of these names same as the GPU one. Any new bucket name + SA name requires reapplying all the permissions that were given to the GPU ones separately and adds too many steps for someone to try this out.

Reply:
Yeah, thats NP. But I will say, there will be bucket naming collision if there are 2 separate benchmarking clusters on the same project.

Reply:
Par discussion, for this piece ease of use is more important than supporting multiple clusters with the same tf script. +1 for adjusting this to be the same as GPU story :)

Reply:
For sure! This should already be updated in the latest changes